ecoscope/analysis/classifier.py

```diff
@@ -25,7 +26,16 @@
 
 # pass in a dataframe and output a series
 def apply_classification(
-    dataframe, input_column_name, output_column_name=None, labels=None, scheme="natural_breaks", **kwargs
+    dataframe,
+    input_column_name,
+    output_column_name=None,
+    labels=None,
+    scheme="natural_breaks",
+    label_prefix="",
+    label_suffix="",
+    label_ranges=False,
+    label_decimals=1,
+    **kwargs,
 ):
     """
     Classifies the data in a GeoDataFrame column using specified classification scheme.
```

Comment:
"Classifies the data in a DataFrame column using specified classification scheme."